src/sc_crawler/inspector.py

```
@@ -754,45 +828,68 @@ def inspect_server_benchmarks(server: "Server") -> List[dict]:
+                        framework=framework_path,
+                        benchmark_id=":".join([framework, measurement]),
+                        extend_environment={
+                            "database_engine_version": database_engine_version
+                        },
+                    ),
+                    "config": {"concurrency": "peak"},
+                    "score": data["score"],
+                    "note": f"Latency: {data['latency_avg_ms']} ms, concurrency: {data['peak_concurrency']}.",
+                },
+            ]
     except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)